WBooks/Extensions/InputFields.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  InputFields.swift
+//  WBooks
+//
+//  Created by Juan Silva on 18/06/2021.
+//
+
+import UIKit
+
+@IBDesignable
+open class CustomInputFields: UITextField {
```

Comment:
`open` access level is used when you need to extend the class from another module. Here, we could declare this as `internal` and it would be okay. More info in this [link](https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html#ID5).